packages/ui-scripts/README.md

```diff
@@ -44,69 +36,76 @@ To clean out built/generated files from a package:
 
 If not specified, `modules` defaults to `es`.
 
-#### `test`
+To build (webpack) a package to be consumed as an application:
 
-To run tests for a package:
+`yarn ui-scripts bundle`
 
-`yarn ui-scripts test`
+#### bump
 
-#### `lint`
+To update all package versions
+(defaults to determining the version automatically using commit messages):
 
-To lint (eslint/stylelint) all files:
+`yarn ui-scripts bump [version|major|minor|patch]`
 
-`yarn ui-scripts lint`
+#### clean
 
-To lint specific files:
+To clean out built/generated files from a package:
 
-`yarn ui-scripts lint ./somefile.ts ./someotherfile.css`
+`yarn ui-scripts clean`
 
-To fix lint issues:
+#### deprecate
 
-`yarn ui-scripts lint --fix`
+To deprecate all packages (optional arguments: version, fix version):
```

Comment:
Aren't these required argument? `deprecate.js line62`

Reply:
yep, I've updated the code and description